src/lsm/groove.zig

```diff
@@ -485,16 +486,17 @@ pub fn GrooveType(
             var objects_cache = try ObjectsCache.init(allocator, .{
```

Comment:
I think here we might want to check that runtime options are compatible with comptime ones?  That limit <= max